apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -636,11 +635,11 @@ export function buildMachine(
               await auth.endCardlessVoterSession(
                 constructAuthMachineState(context.workspace)
               );
+              context.workspace.store.setIsPatDeviceConnected(false);
```

Comment:
Kind of a tricky situation - resetting connection status when the state machine resets is required so a shared PAT device (uncommon, but possible if the precinct supplies one) is recalibrated for each voter. It's possible a jam would happen before the ballot is printed, and therefore the same voter would need to go through device calibration twice. I think we should err toward over-calibration rather than under-calibration.